facia-tool/public/js/models/collections/article.js

```diff
@@ -862,8 +865,9 @@ define([
 
                 self.updateEditorsDisplay();
             })
-            .catch(function() {
+            .catch(function(ex) {
                 self.meta.headline(undefined);
+                reportErrors(ex);
```

Comment:
I'm not sharing frontend reportErrors, facia-tool has its own version which is rather dumb.